### PR TITLE
bug No. 1170667 Bug title :red warning "not for your platform" flashes

### PR DESCRIPTION
**Pull request by ankursmooth**
fixed on basis of changes recommended by mstriemer on bugzilla.
"You should be able to set `notices = []` in [1] if the app's slug in `loading` (which comes from [2]).

> {% set notices = [] if app.slug == 'loading' else app_notices(app) %}

[1] https://github.com/mozilla/fireplace/blob/65a024b449d981e9019f1eabbf2ed4f70491a45a/src/templates/_macros/app_tile.html#L71
[2] https://github.com/mozilla/fireplace/blob/65a024b449d981e9019f1eabbf2ed4f70491a45a/src/media/js/views/app.js#L35"

### CODE REVIEW

**Comment by ngokevin** [comment]
Awesome! One small change I would make is to set ```app.loading``` to ```true``` at https://github.com/mozilla/fireplace/blob/master/src/media/js/views/app.js#L22 and use that instead of the slug. Just to be perfect since there could conceivably be a real app with the slug ```loading```.

**Comment by ankursmooth** [comment]
@ngokevin i dont know how to edit this pull request or make changes to it. i have created a new PR #1466 . please see to it and suggest other changes.

**Comment by ankursmooth** [comment]
i have now set loading to true after https://github.com/mozilla/fireplace/blob/master/src/media/js/views/app.js#L22 and used that in app_tile html. sorry for so many commits to this PR.
please check and tell about these changes.
thanks

File: src/templates/_macros/app_tile.html [hunk 1]
```diff
@@ -68,7 +68,7 @@
         'slug': app.slug
       }) }}
     {% endif %}
-    {% set notices = app_notices(app) %}
+    {% set notices =  [] if app.loading == true else app_notices(app) %}
```
Comment (mstriemer) [inline src/templates/_macros/app_tile.html]: No need to compare to `true` just use: `[] if app.loading else app_notices(app)`.

**Comment by ankursmooth** [comment]
took @mstriemer suggestions and committed accordingly.
remove comparison to true of app.loading in src/templates/_macros/app_tile.html

**Comment by mstriemer** [comment]
Looks good you can use `git commit --amend` to avoid creating a new commit for small changes in review. If you've already got several commits you can run `git rebase master -i` to bring your branch up to date with the latest mater and squash your commits into one.

Seeing as there's a commit from Kevin in here I'll do the rebase myself since it will probably be tricky.

Thanks for the patch!

**Comment by ankursmooth** [comment]
Thanks for your help and persons like you makes me feel so good of the open source community.
clearly i am a newbie in these things.

**Pull request closed without merging**